src/Microsoft.Azure.WebJobs.Host/Queues/Bindings/QueueAttributeBindingProvider.cs

```diff
@@ -70,6 +70,10 @@ internal class QueueAttributeBindingProvider : IBindingProvider
             }
 
             IStorageAccount account = await _accountProvider.GetStorageAccountAsync(context.Parameter, context.CancellationToken, _nameResolver);
+            if (account == null)
```

Comment:
If we're adding this validation, it would be good to do it consistently across all the other bindings.